### PR TITLE
Added cache configuration for preview & websocket

### PR DESCRIPTION
**Pull request by chirimoya**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | none
| License | MIT
| Documentation PR | none

#### What's in this PR?

Cache configuration for preview & web socket.

#### Why?

If you have multiple web servers you need a central cache for the preview and web socket connection.

#### Example Usage

~~~yml
sulu_preview:
    cache:
        type: redis

sulu_websocket:
    cache:
        type: redis
~~~

### CODE REVIEW

File: src/Sulu/Bundle/PreviewBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -45,9 +45,79 @@ public function getConfigTreeBuilder()
...
             ->end()
         ->end();
 
         return $treeBuilder;
     }
+
+    /**
+     * @param string $name
+     *
+     * @return \Symfony\Component\Config\Definition\Builder\NodeDefinition
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/PreviewBundle/DependencyInjection/Configuration.php]: `@return NodeDefinition` and short description

File: src/Sulu/Bundle/WebsocketBundle/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -38,8 +38,78 @@ public function getConfigTreeBuilder()
...
+                ->end()
             ->end();
 
         return $treeBuilder;
     }
+
+    /**
+     * @param string $name
+     *
+     * @return \Symfony\Component\Config\Definition\Builder\NodeDefinition
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/WebsocketBundle/DependencyInjection/Configuration.php]: same here (:

**Comment by chirimoya** [comment]
@wachterjohannes please have a look at the `SuluTestKernel` & `composer.json`

File: composer.json [hunk 3]
```diff
@@ -19,6 +19,8 @@
         "doctrine/orm": "2.5.*",
         "doctrine/phpcr-bundle": "~1.2.4",
         "doctrine/phpcr-odm": "~1.3.0",
+        "doctrine/doctrine-bundle": "~1.0",
```
Comment (wachterjohannes) [inline composer.json]: i would prefer ^1.0

Comment (chirimoya) [inline composer.json]: needs to bee discussed

**Pull request merged**